### DISCUSSION TITLE
Integrate Webdriver BiDi into Cypress?

### DISCUSSION THREAD

**Discussion by mirobo** [cypress-io/cypress#29429] [General]
Will Cypress try to integrate Webdriver BiDi as a proof of concept at some time? Reading the goals of the Webdriver BiDi project I see a lot of potential for all automation tools (including Cypress of source). And the framework that encapsulates Webdriver BiDi with a very good "higher level" API like Cypress currently has, might be the most successful in long term.

https://developer.chrome.com/blog/webdriver-bidi

https://github.com/w3c/webdriver-bidi/blob/main/roadmap.md

**Comment by surfmuggle** [comment]
@mirobo i assume they will based on this issue [Migrate Firefox from CDP to WebDriver BiDi ](https://github.com/cypress-io/cypress/issues/29714)

**Reply by mirobo** [reply]
> @mirobo i assume they will based on this issue [Migrate Firefox from CDP to WebDriver BiDi ](https://github.com/cypress-io/cypress/issues/29714)

Thanks for the hint, I'll close the discussion! If there are performance benefits, I'm sure they will also migrate chromium-based automation to WebDriver BiDi :-)
